### PR TITLE
Dev minor fix

### PR DESCRIPTION
**Pull request by edwinnglabs**
## Description

1. fix multiplot color
2. fix docstring

### CODE REVIEW

File: orbit/diagnostics/plot.py [hunk 1]
```diff
@@ -556,8 +556,8 @@ def plot_bt_predictions(bt_pred_df, metrics=smape, split_key_list=None,
         figsize = (16, 8)
 
     metric_vals = bt_pred_df.groupby('split_key').apply(lambda x:
-                                                        metrics(x[x['training_data'] == False]['actuals'],
-                                                                x[x['training_data'] == False]['prediction']))
+                                                        metrics(x[x['training_data']]['actuals'],
```
Comment (wangzhishi) [inline orbit/diagnostics/plot.py:559]: you want to use `~x['training_data']`?

Comment (edwinnglabs) [reply orbit/diagnostics/plot.py:559]: my bad

File: orbit/diagnostics/plot.py [hunk 2]
```diff
@@ -556,8 +557,8 @@ def plot_bt_predictions(bt_pred_df, metrics=smape, split_key_list=None,
         figsize = (16, 8)
 
     metric_vals = bt_pred_df.groupby('split_key').apply(lambda x:
-                                                        metrics(x[x['training_data'] == False]['actuals'],
-                                                                x[x['training_data'] == False]['prediction']))
+                                                        metrics(~x[x['training_data']]['actuals'],
```
Comment (wangzhishi) [inline orbit/diagnostics/plot.py:560]: ah...somehow it is still not quite right? should be `x[~x['training_data']`?

**Review by wangzhishi — APPROVED** [review]
LGTM

**Pull request merged**